Enhance CMSSlider component with improved accessibility features

viratatwebflow opened the pull request:
- Add New CMS Map Example
- Added ARIA attributes to the CMSSlider and SlideItem components to improve accessibility for screen readers.
- Updated the keyboard navigation logic to ensure a better user experience for keyboard users.

viratatwebflow committed:
Enhance CMSSlider component with improved accessibility features

- Added ARIA attributes to the CMSSlider and SlideItem components to improve accessibility for screen readers.
- Updated the keyboard navigation logic to ensure a better user experience for keyboard users.

zplata commented on cms-map/README.md:
```diff
@@ -0,0 +1,105 @@
+
+## React + TypeScript + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```
Can we tweak the README to align similar to the other examples in the repo (getting it up and running in Webflow, removing boilerplate README details)

viratatwebflow replied:
Done!

zplata commented on cms-map/.env.example:
```diff
@@ -0,0 +1 @@
+VITE_MAP_KEY=your_mapbox_api_key_here.example
```
Any disclaimers we need to make with this API key on client side?

viratatwebflow replied:
This is public key safe to expose on client side. I'll add info folks need to setup the mapbox account

zplata reviewed (approved):
Lgtm! Let's just make sure README is aligned with others but super excited to get this one in

viratatwebflow committed:
Update README.md for CMS Map component with enhanced features and setup instructions

- Revised the README to provide a clearer overview of the CMS Map component, including its integration with Mapbox and Webflow CMS.
- Added detailed setup instructions, component properties, and technical implementation notes.
- Included a preview screenshot and highlighted key features for better user understanding.

viratatwebflow committed:
Updated README

viratatwebflow committed:
Added CMS Map section to README with detailed features and integration information
